earthaccess/daac.py

```diff
@@ -148,6 +149,7 @@ def find_provider(
 
 
 def find_provider_by_shortname(short_name: str, cloud_hosted: bool) -> Union[str, None]:
+    """Placeholder."""
```

Comment:
Please remove the docstrings that contain _only_ "Placeholder". We were using that to fix up docstrings that contained other things (like parameter descriptions) but no subject line. For these undocumented functions, the correct fix is to write a real docstring, but we can consider that out of scope for this PR.

Additionally, if we put placeholder text, we can't use the Ruff D1 rules to catch functions, etc. without docstrings!

Reply:
Sorry about that! forgot about your last comment on ignoring placeholder for functions with no docstrings.

Reply:
No sweat :)